backend/cfg/llvmize.ml

```diff
@@ -1221,6 +1451,11 @@ let cfg (cl : CL.t) =
       } =
     cfg
   in
+  let fun_has_try =
+    DLL.exists layout ~f:(fun label ->
+        (Label.Tbl.find blocks label).is_trap_handler)
```

Comment:
The check is not sensitive to layout, it would be nicer to traverse the block directly.